gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.11.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.3-bin.zip
```

Comment:
@afischerdev It's best to use the latest official recommendation, currently Gradle `8.13`:
https://developer.android.com/build/releases/gradle-plugin

Also it's better to upgrade Gradle using command line so all Gradle files are updated correctly:
(shell scripts `gradlew` and `gradlew.bat` and the `gradle-wrapper.jar`)
```groovy
./gradlew wrapper --gradle-version 8.13
```